Composer/packages/lib/ui-shared/src/components/DisplayReadme/DisplayReadme.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { ScrollablePane } from 'office-ui-fabric-react/lib/ScrollablePane';
+import formatMessage from 'format-message';
+import ReactMarkdown from 'react-markdown';
+
+import { DialogTypes, DialogWrapper } from '../DialogWrapper';
+
+type DisplayReadmeProps = {
+  title: string;
+  readme: string;
```

Comment:
nit: in addition to comment above, readme -> content

Reply:
resolved